### ISSUE TITLE
CI is broken for Coveralls

### ISSUE THREAD

**Issue by plowman** [python-restx/flask-restx#52]
### **Repro Steps**
1. Create a PR
2. The tests run nicely in each environment
3. But suddenly, something goes wrong...

### **Expected Behavior**
The tests, coveralls, etc. should all pass.

### **Actual Behavior**
Coveralls does not pass

### **Error Messages/Stack Trace**
You can see an example failure [here](https://github.com/python-restx/flask-restx/pull/35/checks?check_run_id=437142210).
```
Not on TravisCI. You have to provide either repo_token in .coveralls.yml or set the COVERALLS_REPO_TOKEN env var.
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages/coveralls/cli.py", line 61, in main
    service_name=options['--service'])
  File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages/coveralls/api.py", line 58, in __init__
    self.ensure_token()
  File "/opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages/coveralls/api.py", line 67, in ensure_token
    self.config_filename))
coveralls.exception.CoverallsException: Not on TravisCI. You have to provide either repo_token in .coveralls.yml or set the COVERALLS_REPO_TOKEN env var.
##[error]Process completed with exit code 1.
```

### **Additional Context**
* The probably fix here is to create a Github encrypted secret to match the `COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}` line in test.yml
* This is blocking the tests from passing on #35.

**Comment by ziirish** [comment]
We are aware of this situation. This is a regression due to our change from travis-ci to github actions.

We have a workaround: disable coverage report for forks/PR.

It seems codecov is working on a solution so we are maybe going to switch to codecov once their solution is prodded.
We are actively tracking this issue: https://github.com/codecov/codecov-action/issues/29

**Comment by SVilgelm** [comment]
Temporary fix #44

**Comment by SVilgelm** [comment]
The temporary fix is merged. I'm closing this issue
